dspace-api/src/main/java/org/dspace/authorize/ResourcePolicyServiceImpl.java

```diff
@@ -104,13 +104,22 @@ public ResourcePolicy create(Context context) throws SQLException
     }
 
     @Override
-    public List<ResourcePolicy> find(Context c, DSpaceObject dso, Group group, int action, int notPolicyID) throws SQLException {
-        return resourcePolicyDAO.findByTypeIdGroupAction(c, dso, group, action, notPolicyID);
+    public List<ResourcePolicy> find(Context c, DSpaceObject dso, Group group, int action) throws SQLException {
+        return resourcePolicyDAO.findByTypeGroupAction(c, dso, group, action);
     }
     
+    // TODO: @Override
```

Comment:
Not sure I understand the purpose to this TODO?  Did you mean to add an `@Override` here?